modules/monitoring-enabling-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * monitoring/user-workload-monitoring.adoc
+
+[id="enabling-user-workload-monitoring_{context}"]
+= Enabling user workload monitoring
+
+You can enable user workload monitoring by setting the ` techPreviewUserWorkload/enabled` flag in the cluster monitoring ConfigMap.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Start editing the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Set the `techPreviewUserWorkload` setting to `true` under `data/config.yaml`:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    techPreviewUserWorkload:
+      enabled: true
+----
+
+. Save the file to apply the changes. The user workload monitoring is enabled automatically.
+
+. Optionally, you can check that the user workload monitoring pods were created:
++
+----
+$ oc -n openshift-user-workload-monitoring get pod
+NAME READY STATUS RESTARTS AGE
```

Comment:
Should have a good alignment, example
` oc -n openshift-user-workload-monitoring get pod
NAME                                                      READY   STATUS    RESTARTS   AGE
prometheus-operator-67f8b97b6c-598wx   1/1          Running     0                   3h55m
prometheus-user-workload-0                     5/5          Running     1                   3h55m
prometheus-user-workload-1                     5/5          Running     1                   3h55m
`